a11y-multiple-page-checker/github-action/src/payloadHandler.js

```diff
@@ -13,53 +17,56 @@ export const handle = async path => {
     const event = JSON.parse(data);
 
     const body = init(event);
-    const baseUrl = "https://axe-distributed-action-hanuv4jn2q-uc.a.run.app/?url=" + body.deployment.payload.web_url;
-    
+    const baseUrl =
+      "https://axe-distributed-action-hanuv4jn2q-uc.a.run.app/?url=" +
+      body.deployment.payload.web_url;
+
```

Comment:
my editor was a bit grumpy about the original formating and wouldn't save the file without tweaking it a bit.